modules/distr-tracing-tempo-install-tempomonolithic-web-console.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+//* observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="distr-tracing-tempo-install-tempomonolithic-web-console_{context}"]
+= Installing a TempoMonolithic instance by using the web console
+
+:FeatureName: The TempoMonolithic instance
+include::snippets/technology-preview.adoc[]
+
+You can install a TempoMonolithic instance from the *Administrator* view of the web console.
+
+.Prerequisites
+
+* You are logged in to the {product-title} web console as a cluster administrator with the `cluster-admin` role.
+
+* For {product-dedicated}, you must be logged in using an account with the `dedicated-admin` role.
+
+.Procedure
+
+. Create a project of your choice for the *TempoMonolithic* instance that you will create in a subsequent step: go to *Home* -> *Projects* -> *Create Project*.
+
+. Decide which type of supported storage you will use for storing traces: in-memory storage, a Persistent Volume, or object storage.
```

Comment:
```suggestion
. Decide which type of supported storage you to use for storing traces: in-memory storage, a Persistent Volume, or object storage.
```

Reply:
Thank you. I will apply this suggestion.